code/game/objects/items/tongs.dm

```diff
@@ -0,0 +1,102 @@
+#define TONG_CLACK_CD (2 SECONDS)
```

Comment:
suggestion: make this a variable instead so i can varedit tongs to have a very low cooldown so i can clackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclackclack

you could also not do this but i would be happy if you did